### PR TITLE
Update Readme and community_sharings.rst for 2.0 release

### PR DESCRIPTION
**Pull request by scarlett2018**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -301,45 +301,21 @@ You can use these commands to get more information about the experiment
...
-  * **Blog (in Chinese)** - [AutoML tools (Advisor, NNI and Google Vizier) comparison](http://gaocegege.com/Blog/%E6%9C%BA%E5%99%A8%E5%AD%A6%E4%B9%A0/katib-new#%E6%80%BB%E7%BB%93%E4%B8%8E%E5%88%86%E6%9E%90) by [@gaocegege](https://github.com/gaocegege) - 总结与分析 section of design and implementation of kubeflow/katib
-  * **Blog (in Chinese)** - [A summary of NNI new capabilities in 2019](https://mp.weixin.qq.com/s/7_KRT-rRojQbNuJzkjFMuA) by @squirrelsc
+## **Rleases and Contributing**
+NNI has a monthly release cycle (major releases). Please let us know if you encounter a bug by [filling an issue](https://github.com/microsoft/nni/issues/new/choose).
+
+We apprecicate all contributions. If you are planning to contribute any bug-fixes, please do so without further discussions.
+
+If you plan to contribute new features, new tuners, new training services, etc. please first open an issue or reuse an exisiting issue, and discuss the feature with us. We will discuss with you on the issue timely or set up conference calls if needed.
+
+To learn more about making a contribution to NNI, please refer to our [How-to contribution page](https://github.com/microsoft/nni/wiki/How-to-contribute).
```
Comment (kvartet) [inline README.md:311]: maybe this [page](https://nni.readthedocs.io/en/v2.0/contribution.html) is more helpful.

File: docs/en_US/CommunitySharings/community_sharings.rst [hunk 2]
```diff
@@ -14,3 +16,25 @@ Different from the tutorials and examples in the rest of the document which show
...
+
+External Repositories
+=====================
+   * Run `ENAS <https://nni.readthedocs.io/en/latest/NAS/ENAS.html>`__ with NNI
+   * `Hyperparameter Tuning for Matrix Factorization <https://github.com/microsoft/recommenders/blob/master/examples/04_model_select_and_optimize/nni_surprise_svd.ipynb>`__ with NNI
+   * `scikit-nni <https://github.com/ksachdeva/scikit-nni>`__ Hyper-parameter search for scikit-learn pipelines using NNI
+
+Relevant Articles
+=================
+  * `Hyper Parameter Optimization Comparison <docs/en_US/CommunitySharings/HpoComparison.rst>`__
```
Comment (kvartet) [inline docs/en_US/CommunitySharings/community_sharings.rst:32]: Boken link, please use `Hyper Parameter Optimization Comparison <./HpoComparison.rst>`. Same below.

File: README.md [hunk 3]
```diff
@@ -301,45 +301,21 @@ You can use these commands to get more information about the experiment
...
+NNI has a monthly release cycle (major releases). Please let us know if you encounter a bug by [filling an issue](https://github.com/microsoft/nni/issues/new/choose).
+
+We apprecicate all contributions. If you are planning to contribute any bug-fixes, please do so without further discussions.
+
+If you plan to contribute new features, new tuners, new training services, etc. please first open an issue or reuse an exisiting issue, and discuss the feature with us. We will discuss with you on the issue timely or set up conference calls if needed.
+
+To learn more about making a contribution to NNI, please refer to our [How-to contribution page](https://github.com/microsoft/nni/wiki/How-to-contribute). 
+
+We appreciate all contributions and thanks all the contributors!
+<a href="https://github.com/keras-team/autokeras/graphs/contributors"><img src="https://notes.haifengjin.com/img/contributors.svg" /></a>
```
Comment (kvartet) [inline README.md:314]: why autokeras?

Comment (QuanluZhang) [reply README.md:314]: why autokeras?

File: docs/en_US/CommunitySharings/community_sharings.rst [hunk 4]
```diff
@@ -14,3 +16,25 @@ Different from the tutorials and examples in the rest of the document which show
...
     Use NNI on Google Colab <NNI_colab_support>
     Auto Completion for nnictl Commands <AutoCompletion>
+
+External Repositories and References
+====================================
+With authors' permission, we listed a set of NNI usage examples and relevant articles.
+
+External Repositories
+=====================
+   * Run `ENAS <https://nni.readthedocs.io/en/latest/NAS/ENAS.html>`__ with NNI
```
Comment (ultmaster) [inline docs/en_US/CommunitySharings/community_sharings.rst:26]: I think it's no longer external.

**Comment by scarlett2018** [comment]
comments fixed, if we can make the contributors picture before others PR done, we need to update the file again. if not, we can skip update the contributors section later. @kvartet

**Review by ultmaster — APPROVED** [review]

**Comment by kvartet** [comment]
> comments fixed, if we can make the contributors picture before others PR done, we need to update the file again. if not, we can skip update the contributors section later. @kvartet

Added.

**Review by kvartet — APPROVED** [review]

**Pull request merged**